jax/_src/lax/parallel.py

```diff
@@ -1051,13 +1132,23 @@ def _ppermute_lowering(ctx, x, *, axis_name, perm):
   if is_manual:
     channel = ctx.module_context.new_channel()
     other_args = dict(
-        channel_handle=hlo.ChannelHandle.get(channel, mlir.DEVICE_TO_DEVICE_TYPE))
+        channel_handle=hlo.ChannelHandle.get(
+            channel, mlir.DEVICE_TO_DEVICE_TYPE
+        )
+    )
```

Comment:
Could you undo these unrelated auto-formatting changes?

Reply:
suggestion applied!

Reply:
Google linters can’t hurt you when working on jax code.